source/appendices/e-audit-requirements-and-tests/02-audit-tests.rst

```diff
@@ -5,1045 +5,401 @@
 
-The auditor must force 50 Limit_Exceeded turnaways.
+Where a platform has fewer than 80 items, the auditor MUST make at least one investigation per item and testing should result in 80 Total_Item_Investigations and the item-count of Unique_Item_Investigations.
 
-* The auditor will log into the site. They will select and make a search on a database or browse to a database causing the user limit to reach the maximum allowance. The auditor will then log into the same site using a different computer. The auditor will then repeat the action made on the previous computer (select and make a search on a database or browse to a database).
-* The user should then be refused access as the concurrent/simultaneous-user limit has exceeded. Each time access is refused, the auditor will record this as 1 Limit_Exceeded.
 
-Each of these concurrent/simultaneous access denials must report 1 Limit_Exceeded in the DR_D2 Standard View.
+E.2.4 Title Report Audit Tests: Books and Reference_Works
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""
 
-**Option 3**: The content provider denies the user access when the concurrent/simultaneous-user limit is exceeded upon accessing an item within a database.
+The Title Report will be COUNTER-compliant for book and/or reference work content if the following audit tests are passed. The book-related Standard Views of the Title Report (TR_B1, TR_B2, TR_B3) will be COUNTER-compliant if the metrics match those in the Title Report, with the appropriate aggregation.
 
-The auditor must force 50 Limit_Exceeded turnaways.
+Audit test requirements vary depending on the set up of the platform, as indicated by the Options within the tests below.
 
-* The auditor will log into the site and will navigate to and request an item. This will cause the user limit to reach the maximum allowance.The auditor will log into the site again using a different computer. The auditor will then repeat the action made on the previous computer (navigate to and request an item).
-* After the item has been requested the user should then be denied access. Each time access is refused, the auditor will record this as 1 Limit_Exceeded.
+Title Report audit tests for books and reference works apply to Host_Types Aggregated_Full_Content, eBook, and eBook_Collection.
 
-Each of these concurrent/simultaneous turnaways must report 1 Limit_Exceeded in the DR_D2 Standard View.
+Note that Data_Types Book and Reference_Work are included in the audit tests, as Reference_Works are included in the book-related Standard Views of the Title Report.
 
+E.2.4.1 Unique_Title_Investigations and Unique_Title_Requests
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
 
-Audit Test D2-2: No_License
-...........................
+**Option 1**: Book_Segments and/or Reference_Items are available, users can elect to access Books and/or Reference_Works segment-by-segment.
```

Comment:
I assume this should be the case "Only Book_Segments and/or References_Items are available" since Option 3 covers that case that content is delivered both as whole books and items? The "user can elect" seems to indicate otherwise.

Reply:
I think I was wrong. Option 3 is only about whole book/reference work requests, and this covers the item requests, both in case only items or both whole books and items are available?